### DISCUSSION TITLE
用户使用简单的纸笔计算构建zk-SNARK的手册（来自rebase公众号，我还没有验证）

### DISCUSSION THREAD

**Discussion by FighterAircraft** [zkp-co-learning/zkp-co-learn#90] [Materials]
https://github.com/LeastAuthority/moonmath-manual 

《MoonMath手册》旨在使用户使用简单的纸笔计算构建zk-SNARK。在整个手册中，鼓励用户完成练习，以确保可以参与到zk-SNARK的基础构建块中。在这个过程中，读者可以构建最小的但功能完整的加密堆栈，以计算 zk-SNARK，而不需要计算机的参与

**Comment by Demian101** [comment]
cool
